docs/input-kafka.asciidoc

```diff
@@ -641,7 +678,8 @@ The topics configuration will be ignored when using this configuration.
   * Value type is <<string,string>>
   * Default value is `"org.apache.kafka.common.serialization.StringDeserializer"`
 
-Java Class used to deserialize the record's value
+Java Class used to deserialize the record's value. This settings conflicts with <<plugins-{type}s-{plugin}-schema_registry_url>>.
+A custom value deserializer could be used only if not using a Schema Registry.
```

Comment:
```suggestion
Java Class used to deserialize the record's value. 

A custom value deserializer can be used only if you are not using a Schema Registry.
Use either the value_deserializer_class config option or the
<<plugins-{type}s-{plugin}-schema_registry_url>> config option, but not both.
```

Reply:
Thanks @karenzone for the suggestion, integrated!